lib/room.rb

```diff
@@ -1,12 +1,25 @@
-class Rom
+class Room
+  attr_reader :category, :length, :width
```

Comment:
Since we don't ever call `length` or `width` as methods, you don't need them as attr_readers